tests/unit/Makefile.am

```diff
@@ -5,31 +5,39 @@
 #
 
 if ENABLE_UNIT_TESTS
-AM_CPPFLAGS = -I$(top_srcdir)/include $(CUDA_CPPFLAGS)
-LDADD = $(top_builddir)/src/libinternal_net_plugin.la $(CUDA_LIBS)
-AM_LDFLAGS = $(CUDA_LDFLAGS)
-
+AM_CPPFLAGS = -I$(top_srcdir)/include
+AM_CPPFLAGS += -isystem $(abs_top_srcdir)/3rd-party/nccl/$(DEVICE_INTERFACE)/include
+AM_CPPFLAGS += -isystem $(abs_top_srcdir)/3rd-party/uthash/include
+LDADD = $(top_builddir)/src/libinternal_net_plugin.la
 noinst_HEADERS = test-common.h
 
 noinst_PROGRAMS = \
 	deque \
 	freelist \
 	msgbuff \
-	show_tuner_decisions \
 	scheduler \
 	idpool \
 	ep_addr_list \
 	mr
 
-TESTS = $(noinst_PROGRAMS)
-
-show_tuner_decisions_SOURCES = show_tuner_decisions.c
-show_tuner_decisions_LDADD = $(top_builddir)/src/libinternal_tuner_plugin.la
+if !ENABLE_NEURON
```

Comment:
incorrect?

Reply:
no, neuron does not contain a tuner.h interface file, tuner cannot be built for neuron until they opt in and vend a copy.